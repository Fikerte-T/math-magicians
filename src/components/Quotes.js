import React from 'react';

const Quotes = () => (
      <div className="quote-page">
            <blockquote>Mathematics is not about numbers, equations, computations, or algorithms:
            it is about understanding. - <span className="author">William Paul Thurson</span>
            </blockquote>
      </div>
);

export default Quotes;
