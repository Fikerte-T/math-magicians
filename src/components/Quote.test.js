import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from './Quotes.js';

it('Quotes snapshot test', () => {
  const tree = renderer
    .create(<Quotes />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});