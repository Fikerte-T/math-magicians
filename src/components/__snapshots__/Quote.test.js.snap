// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes snapshot test 1`] = `
<div
  className="quote-page"
>
  <blockquote>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. - 
    <span
      className="author"
    >
      William Paul Thurson
    </span>
  </blockquote>
</div>
`;

exports[`calculator snapshot test 1`] = `
<div
  className="quote-page"
>
  <blockquote>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. - 
    <span
      className="author"
    >
      William Paul Thurson
    </span>
  </blockquote>
</div>
`;
